src/test/resources/migration/custom/schema-v5.2.0-postgresql.sql

```diff
@@ -624,6 +624,7 @@ CREATE TABLE "NOTIFICATIONRULE"
     "NAME" varchar(255) NOT NULL,
     "NOTIFICATION_LEVEL" varchar(255) NULL,
     "NOTIFY_CHILDREN" bool NULL,
+    "LOG_SUCCESSFUL_PUBLISH" bool NULL,
```

Comment:
This file is intended to be used as a baseline when testing the custom migrations from `v5.2.0` to `v5.3.0`: https://github.com/DependencyTrack/hyades-apiserver/tree/main/src/test/java/org/dependencytrack/persistence/migration/change/v530

So should not be modified with changes introduced in v5.3.0 (the current in-progress version).